Add elasticsearch_hosts conf

jkrol2 opened the pull request:
With elasticsearch hosts configuration ES can be configured on other
machine than graylog server

pkrupa2 commented on tasks/configure.yml:
```diff
@@ -46,4 +46,47 @@
 #- name: Create mongodb user
+- name: wait for graylog to configure
+  uri:
+    url: http://localhost:9000
```
there is graylog_address variable specified in defaults/main.yml, it should be used here insead of localhost

pkrupa2 commented on tasks/configure.yml:
```diff
@@ -46,4 +46,47 @@
+  uri:
+    url: http://127.0.0.1:9000/api/system/indices/index_sets
+    user: "soi"
+    password: "soi.098"
```
login credentials should be specified as variables! In tests it should be specified as "admin:password".

Please remove commit with those credentials from GitHub.

pkrupa2 commented on tasks/configure.yml:
```diff
@@ -46,4 +46,47 @@
+
+- name: get index id
+  uri:
+    url: http://127.0.0.1:9000/api/system/indices/index_sets
```
there is graylog_address variable specified in defaults/main.yml, it should be used here insead of localhost

pkrupa2 commented on tasks/configure.yml:
```diff
@@ -46,4 +46,47 @@
+
+- name: configure index retention
+  uri:
+    url: "http://127.0.0.1:9000/api/system/indices/index_sets/{{response.json.index_sets[0].id}}"
```
there is graylog_address variable specified in defaults/main.yml, it should be used here insead of localhost

pkrupa2 commented on tasks/configure.yml:
```diff
@@ -46,4 +46,47 @@
+  uri:
+    url: "http://127.0.0.1:9000/api/system/indices/index_sets/{{response.json.index_sets[0].id}}"
+    user: "soi"
+    password: "soi.098"
```
Credentials!!! Please remove commit with those informations!

Commit:
Add elasticsearch_hosts conf

With elasticsearch hosts configuration ES can be configured on other
machine than graylog server
